pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -92,9 +93,10 @@ func NewControllerV2(ctx context.Context, jobInformer batchv1informers.JobInform
 				Name: "cronjob",
 			},
 		),
+		name:        controllerName,
 		kubeClient:  kubeClient,
 		broadcaster: eventBroadcaster,
-		recorder:    eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: "cronjob-controller"}),
+		recorder:    eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: controllerName}),
```

Comment:
Maybe a better option would be to move the recorder changes to the followup PR and drive the discussion there.